Package.swift

```diff
@@ -11,7 +11,6 @@ let package = Package(
     .executable(name: "Runner", targets: ["Runner"]),
     .executable(name: "Server", targets: ["Server"]),
     .library(name: "Database", targets: ["Database"]),
-    .library(name: "DatabaseTestSupport", targets: ["DatabaseTestSupport"]),
```

Comment:
I accidentally deleted this line, but it's making me think that unless other packages depend on this one, maybe we don't need to declare anything here but public products?